recipes/bzip2/meta.yaml

```diff
@@ -13,10 +13,6 @@ source:
 
 build:
   number: 0
-  features:
-    - vc9                # [win and py27]
-    - vc10               # [win and py34]
-    - vc14               # [win and py35]
```

Comment:
Also please add `python  # [win]` (2 spaces before selector) to the `requirements/build`.